src/core/Provider.ts

```diff
@@ -1,16 +1,12 @@
 import React, { createElement, useRef, useDebugValue } from 'react'
 
-import { AnyAtom, Scope } from './types'
-import { AtomState, State } from './vanilla'
-import {
-  Store,
-  createStore,
-  subscribeToStore,
-  getStoreContext,
-} from './contexts'
+import type { AnyAtom, Scope } from './types'
+import type { AtomState, State } from './vanilla'
+import { createStore, getStoreContext } from './contexts'
 import { useMutableSource } from './useMutableSource'
 
-const getState = (store: Store) => store.s
+const getState = (state: State) => state
+const subscribe = (_state: State, _callback: () => void) => () => {}
```

Comment:
Ah, that reminds me that I need to check the react devtools behavior. Thanks!

Reply:
It turns out that this no-op subscribe doesn't work as expected.
https://codesandbox.io/s/react-typescript-forked-0ukeg?file=/src/App.tsx
It will only update if it renders from outside.

I will fix this, but in a fundamental manner (or totally drop it). It will be a follow-up work in a different branch/PR.